helper_test.go

```diff
@@ -1177,6 +1201,47 @@ func createVariableSetVariable(t *testing.T, client *Client, vs *VariableSet, op
 	}
 }
 
+// Attempts to upgrade an organization to the business plan. Requires a user token with admin access.
+func upgradeOrganizationSubscription(t *testing.T, client *Client, organization *Organization) {
+	req, err := client.newRequest("GET", "admin/feature-sets", featureSetListOptions{
+		Q: "Business",
+	})
+	if err != nil {
+		t.Fatal(err)
+		return
+	}
+
+	fsl := &featureSetList{}
+	err = client.do(context.Background(), req, fsl)
+	if err != nil {
+		t.Fatalf("failed to enumerate feature sets: %v", err)
+		return
+	} else if len(fsl.Items) == 0 {
+		t.Fatalf("feature set response was empty")
+		return
+	}
+
+	opts := updateFeatureSetOptions{
+		RunsCeiling:        10,
```

Comment:
Can we remove this field and leave the default? This can have consequences for testing and feels arbitrary in this context.